src/start/AuthScreen.js

```diff
@@ -234,6 +234,7 @@ class AuthScreen extends PureComponent<Props> {
     const { dispatch, realm } = this.props;
     const auth = webAuth.authFromCallbackUrl(event.url, otp, realm);
     if (auth) {
+      dispatch(resetToLoading());
       dispatch(loginSuccess(auth.realm, auth.email, auth.apiKey));
```

Comment:
There's a couple of other places we dispatch `loginSuccess`. We probably want this there too, right?